code/game/sound.dm

```diff
@@ -80,11 +80,6 @@
 
```

Comment:
while you're in here, there's a line `S.falloff = (falloff ? falloff : FALLOFF_SOUNDS)` that should be replaced with `S.falloff = falloff || FALLOFF_SOUNDS`